internal/api/v1/applications.go

```diff
@@ -167,6 +168,62 @@ func (hc ApplicationsController) Show(w http.ResponseWriter, r *http.Request) AP
 	return nil
 }
 
+func (hc ApplicationsController) ServiceApps(w http.ResponseWriter, r *http.Request) APIErrors {
```

Comment:
Can you add a comment that describes what this endpoint is about?

Reply:
Sure.

Reply:
Done. Commit 1c96529a8810d7ed6c73dac70ab1d8498f212db6